tests/gaming/wesnoth.pm

```diff
@@ -0,0 +1,69 @@
+# Copyright (C) 2020 SUSE LLC
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, see <http://www.gnu.org/licenses/>.
+#
+# Summary: The Battle for Wesnoth game test
+# Maintainer: Christian Lanig <clanig@suse.com>
+
+use base "x11test";
+use strict;
+use warnings;
+use testapi;
+
+sub run {
+    select_console "x11";
+    ensure_installed "wesnoth";
```

Comment:
I assume because of the bigger dataset the installation can be a bit slower at times. Better to add a bigger timeout value here